plugins/test/unit/__init__.py

```diff
@@ -9,6 +10,10 @@
 devel_base.block_load_conf()
 
 
+if django.VERSION >= (1, 9):
+    django.setup()
```

Comment:
If the unittests need it to do what they do, then presumably anything doing similar things to the unittests will also need it. Does it make more sense to put this into an initialization function rather than make the test runtime environment differ from the normal runtime environment?